controllers/user.js

```diff
@@ -130,9 +130,9 @@ export default class UserController {
             message: [error.message]
           }));
       }
-      return res.json({
+      return res.status(404).json({
         success: false,
-        message: 'User not found'
+        message: 'User not found.'
```

Comment:
@leksyib I thought you guys agreed on using `error` here?

Reply:
Yes we did, this was there before we agreed and I'm changing it now.